app/cdap/components/Administration/TetheringTabContent/OdfTetheringConnections/PendingRequests.tsx

```diff
@@ -48,14 +48,17 @@ const PendingRequests = ({ pendingRequests, handleEdit, handleDelete }: IPending
       onEditClick={() => handleEdit(CONNECTION_STATUS, instanceName)}
     />
   );
+  const canSeeReqHistory = false; // Will remove after request history page is ready
 
   return (
     <>
       <HeaderContainer>
         <HeaderTitle>{T.translate(`${I18NPREFIX}.pendingRequestHeader`)}</HeaderTitle>
-        <PendingRequestHistory to="/administration/tethering/requestHistory">
-          {T.translate(`${I18NPREFIX}.pendingRequestHistory`)}
-        </PendingRequestHistory>
+        {canSeeReqHistory && (
```

Comment:
This change is unrelated to the scope of this PR, but since it was a tiny change, I figured I can tailor it to this PR.